### DISCUSSION TITLE
Not getting app_account_token in responseBody.Latest_receipt_info

### DISCUSSION THREAD

**Discussion by Ankit-R** [hyochan/react-native-iap#2421] [General]
Hello,

I am not getting app_account_token in responseBody.Latest_receipt_info. I am passing it while executing requestPurchase
```typescript
RNIap.requestPurchase({
    sku: 'some_product_id',
    andDangerouslyFinishTransactionAutomaticallyIOS: false,
    appAccountToken: 'some_token'
  })
``` 

"app_account_token" is missing in "latest_receipt_info" payment receipt.
We should be getting this in "latest_receipt_info" according to [this](https://developer.apple.com/documentation/appstorereceipts/responsebody/latest_receipt_info)
react-native-iap version: 12.10.5

Can someone help?

If anyone else is facing this issue then I'll post this in github issues

**Comment by franbe7** [comment]
Hey @Ankit-R! Could you solve this issue? The same is happening to me.

**Comment by textual** [comment]
looks like youre sending appAccountToken properly. are you retrieving transactionInfo.appAccountToken or something else?

**Comment by ahmedrowaihi** [comment]
is this the reason? it must be in UUID format ?
[developer.apple-appaccounttoken](https://developer.apple.com/documentation/appstoreserverapi/appaccounttoken)

![image](https://github.com/dooboolab-community/react-native-iap/assets/67356781/5d5abaee-790c-4b94-b715-3fb9e560c740)
